src/python/grpcio/grpc/_cython/_cygrpc/call.pyx.pxi

```diff
@@ -37,13 +37,19 @@ cdef class Call:
     self.c_call = NULL
     self.references = []
 
-  def start_batch(self, operations, tag):
+  def start_batch(self, operations, tag, client_side=False):
```

Comment:
Break this up into `_start_batch(self, operations, tag, retain_self)`, `start_client_batch(self, operations, tag)`, and `start_server_batch(self, operations, tag)`.